src/test/java/org/sagebionetworks/bridge/spring/controllers/CRCControllerTest.java

```diff
@@ -1439,7 +1571,7 @@ private String makeAppointment(String identifier, AppointmentStatus status) {
         return FHIR_CONTEXT.newJsonParser().encodeResourceToString(appt);
     }
     
-    private String makeProcedureRequest() { 
+    private String makeProcedureRequest() {
         ProcedureRequest procedure = new ProcedureRequest();
         
         Identifier id = new Identifier();
```

Comment:
Some tests that appear to be missing:

internalLabShipmentRequest()
    - account not found test
    - ship requests limited to one
    
validateAndGetAddress()
    - missing first name test
    - missing last name test
    - missing address1 test
    - missing state test
    - missing zip code test
    - test if phone is invalid
    - no test of getLabShipmentStatus